Omaha build fail after installing 10.0.18362 Windows SDK

We have to install `10.0.18362` because that SDK became a requirement for C77. (https://chromium.googlesource.com/chromium/src/+/d99198809d972d29323d71b347ab96e82f6a425a%5E%21/#F0)

Omaha build failed after installing it.
```
________Compiling resource scons-out\dbg-win\obj\mi_exe_stub\mi_generated_resources_zh-TW.res
Microsoft (R) Windows (R) Resource Compiler Version 10.0.10011.16384
Copyright (C) Microsoft Corporation.  All rights reserved.

________Precompiling scons-out\dbg-win\obj\base\base_pch.pch
cl : Command line warning D9035 : option 'Yd' has been deprecated and will be removed in a future release
precompile.cc
C:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um\VersionHelpers.h(162): error C2220: warning treated as error - no 'object' file generated
C:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um\VersionHelpers.h(162): warning C4800: 'BOOL': forcing value to bool 'true' or 'false' (performance warning)
scons: *** [scons-out\dbg-win\obj\base\base_pch.pch] Error 2
scons: building terminated because of errors.

```

--- a/omaha/hammer-brave.bat
+++ b/omaha/hammer-brave.bat
@@ -5,7 +5,7 @@
 set OMAHA_PSEXEC_DIR=%ProgramFiles(x86)%\pstools
 
 :: Set VS environment variables.
-call "%VS140COMNTOOLS%..\..\VC\vcvarsall.bat"
+call "%VS140COMNTOOLS%..\..\VC\vcvarsall.bat" x86 10.0.10586.0
 
 setlocal
 

